core/templates/pages/subtopic-viewer-page/navbar-breadcrumb/subtopic-viewer-navbar-breadcrumb.directive.html

```diff
@@ -1,10 +1,7 @@
-<div class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
-  <span class="oppia-navbar-breadcrumb-separator"></span>
-  <span>
-    <a ng-href="<[getTopicUrl()]>" style="color: white;"><[topicName]></a>
-  </span>
-  <span class="oppia-navbar-breadcrumb-separator"></span>
-  <span>
-    <[subtopicTitle]>
-  </span>
+<div ng-if="subtopicTitle" class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak" style="padding-top: 0; align-items: center; height: 56px;">
```

Comment:
Use classes here and below. Avoid inline styling.

Reply:
Done.